package.json

```diff
@@ -96,14 +96,20 @@
             "markdownDescription": "The path of the folder containing the Swift executables. The default is to look in **$PATH**.",
             "order": 1
           },
+          "swift.runtimePath": {
+            "type": "string",
+            "default": "",
+            "description": "The path of the folder containing the Swift runtime libraries. This will only affect runtime behavior when libraries are not discovered by RPath.",
+            "order": 2
+          },
           "swift.buildArguments": {
             "type": "array",
             "default": [],
             "items": {
               "type": "string"
             },
             "markdownDescription": "Arguments to pass to `swift build`. Keys and values should be provided as separate entries. If you have created a copy of the build task in `tasks.json` then these build arguments will not be propogated to that task.",
-            "order": 2
+            "order": 11
```

Comment:
You can probably undo all the order setting as this will go into the advanced settings subsection as discussed in the sdk PR